docs/csharp/programming-guide/classes-and-structs/codesnippet/CSharp/polymorphism_1.cs

```diff
@@ -49,16 +49,18 @@ static void Main(string[] args)
             // can all be used whereever a Shape is expected. No cast is
             // required because an implicit conversion exists from a derived 
             // class to its base class.
-            System.Collections.Generic.List<Shape> shapes = new System.Collections.Generic.List<Shape>();
-            shapes.Add(new Rectangle());
-            shapes.Add(new Triangle());
-            shapes.Add(new Circle());
+            var shapes = new System.Collections.Generic.List<Shape>
```

Comment:
Would it make sense to also simplify `System.Collections.Generic.List<Shape>` to just `List<Shape>`?

Reply:
I totally agree with you ! Furthermore, I would suggest we include the using statement,
using System.Collections.Generic, at the top of our source file, in order when someone copy and paste it in an environment, in order to test it not to forget it.

Reply:
In that case, I think that `using System;` (required for `Console`) should be added as well.